cypress/integration/userCanAccessPaymentForm.spec.js

```diff
@@ -0,0 +1,51 @@
+describe('User can access payment form', () => {
+  beforeEach(() => {
+    cy.server()
+    cy.route({
+      method: 'POST',
+      url:'http://localhost:3000/auth',
+      response: 'fixture:successful_user_signup.json'
+    })
+    cy.visit('http://localhost:3001')
+  })
+
+  it('Successfully get payment form', () => {
+    cy.get('#signup-button').click()
+    cy.get('#signup-form').within(()=> {
+      cy.get('#nickname-input').type('nickname')
+      cy.get('#name-input').type('name')
+      cy.get('#city-input').type('city')
+      cy.get('#country-input').select('Sweden')
+      cy.get('#email-input').type('user@mail.com')
+      cy.get('#password-input').type('password')
+      cy.get('#password-confirmation').type('password')
+    })
+    cy.get('#submit-signup-form').click()
+    cy.get('#paymentform-button').click()
+    cy.get('#subscribe-button').click()
+    cy.route({
```

Comment:
why do you not stub this request out in the before block

Reply:
and why are you stubbing out successful_subscription_payment if you are not actually paying the subscription in this test

Reply:
Very valid questions, and I'm stupid and reading through the test I'm not really sure wtf I was doing so I removed bunch of unecessary stuff and cleaned it up :)